server/src/database.ts

```diff
@@ -97,6 +97,30 @@ createTable("users", table => {
         table.boolean("approvalRequired").notNullable().defaultTo(true);
         table.text("owner").notNullable();
     });
+}).then(() => {
```

Comment:
nit: i think the enum should be defined at the top of the file, not in the function